### ISSUE TITLE
bug: failed with latest langchain

### ISSUE THREAD

**Issue by gaocegege** [litanlitudan/skyagi#93]
```
LLM initialization check failed: 1 validation error for ChatOpenAI
__root__
  Parameters {'model'} should be specified explicitly. Instead they were passed in as part of `model_kwargs` parameter. (type=value_error)
```

**Comment by gaocegege** [comment]
I am trying the latest langchain because I met https://github.com/hwchase17/langchain/issues/4086

**Comment by qizheng7** [comment]
@gaocegege yeah, that's why I hard-coded the langchain version because it breaks the compatibility across versions too often. Will need to look at what changed again this time. sigh.....

**Comment by tonyz0x0** [comment]
I tried to bump up the langchain version in https://github.com/litanlitudan/skyagi/tree/kejiez/langchain-version and doesn't see the error anymore, @gaocegege you can give a try locally. The `kejiez/langchain-version` branch is based on an ongoing PR: https://github.com/litanlitudan/skyagi/pull/87. Once that PR is merged, then we can make a PR for https://github.com/litanlitudan/skyagi/tree/kejiez/langchain-version
